variables.tf

```diff
@@ -104,3 +104,13 @@ variable "key_vault_id" {
   description = "Key Vault ID"
   default     = {}
 }
+
+variable "mssql_defender_state" {
+  description = "Manages Microsoft Defender state on the mssql server"
+  type        = string
+  default     = "Disabled"

```

Comment:
set it to null by default, we will use "Disabled" within our infrastructure

Reply:
done